This patch brings a critical line up to date from cvs so that
ipc-1.0.6 will compile (09 Jun 2004 agriffis)

http://www.mozdev.org/source/browse/protozilla/ipc/

--- ipc/src/nsPipeChannel.cpp.agriffis	2004-05-17 02:44:43.000000000 -0400
+++ ipc/src/nsPipeChannel.cpp	2004-06-09 13:48:51.146277360 -0400
@@ -210,7 +210,7 @@
     if (NS_FAILED(rv)) return rv;
 
     nsXPIDLCString contentType;
-    rv = MIMEService->GetTypeFromURI(url, contentType);
+    rv = MIMEService->GetTypeFromURI(url, getter_Copies(contentType));
     if (NS_SUCCEEDED(rv) && contentType) {
       mContentType.Assign(contentType);
     }
